lib/DxilPIXPasses/DxilDbgValueToDbgDeclare.cpp

```diff
@@ -1331,9 +1311,6 @@ void VariableRegisters::PopulateAllocaMap_StructType(
       // size would be lost
       PopulateAllocaMap(OffsetAndMember.second);
     } else {
-      assert(m_Offsets.GetCurrentAlignedOffset() ==
```

Comment:
I'm nervous about removing this assert. If GetCurrentAlignedOffset is wrong, can it be fixed to be right? I fear that if these are different, then we won't see any issues until data has made it all the way to the PIX UI.

Reply:
Synced up offline with Jeff, the aligned offsets are correct but are being compared to the packed offsets which don't match if the aligned bits are not the same as the member size. I re-added the assert back in and added a check for whether the size and alignment match instead of modifying SortMembers to return the aligned offsets, maybe we can revisit the aligned/packed code flow when we added more test coverage